test/tests/functional/pbs_acl_host_moms.py

```diff
@@ -52,40 +52,40 @@ def setUp(self):
 
         TestFunctional.setUp(self)
 
-        if len(self.moms) != 2:
-            self.skip_test('test requires two MoMs as input, ' +
-                           '  use -p moms=<mom1>:<mom2>')
+        if len(self.moms) != 1:
+            self.skip_test('test requires one MoM as input, ' +
+                           '  use -p moms=<mom>')
```

Comment:
Awesome! One last thing before I'm happy with this, could you please run this test suite WITHOUT providing any mom/server/client hosts via -p option and attach the logs to the ticket to show that this testsuite will pass by skipping the tests which require the multi-host environment? This is important because we don't want this test to fail if one were to run it without the correct environment.